backend/conduit/profile/views.py

```diff
@@ -10,6 +10,7 @@
 
 from .serializers import profile_schema
 from conduit.tags.serializers import tags_schemas
+from conduit.user.serializers import followers_schema
```

Comment:
Make sure you pull from staging branch and I can merge your code in. Awesome work 👍

Reply:
I did pull from the staging branch.